src/SignalR/clients/csharp/Client/test/FunctionalTests/Hubs.cs

```diff
@@ -45,6 +45,8 @@ public string GetCallerConnectionId()
 
         public ChannelReader<int> StreamEchoInt(ChannelReader<int> source) => TestHubMethodsImpl.StreamEchoInt(source);
 
+        public IAsyncEnumerable<int> StreamIAsyncConsumer(IAsyncEnumerable<int> source) => TestHubMethodsImpl.StreamIAsyncConsumer(source);
```

Comment:
nit:

```suggestion
        public IAsyncEnumerable<int> StreamIAsyncConsumer(IAsyncEnumerable<int> source) => source;
```

And delete TestHubMethodsImpl. StreamIAsyncConsumer.

Reply:
Good catch. This method on all the hubs can be simplified now.